core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/impl/BasePackageServiceImpl.java

```diff
@@ -155,7 +159,8 @@ public PackageInfo getPackageInfo(final ResourceResolver resourceResolver, final
         } else {
             actualPaths = packageModel.getPaths().stream()
                     .filter(s -> resourceResolver.getResource(s.getPath()) != null)
-                    .map(path -> getActualPath(path.getPath(), path.isExcludeChildren(), resourceResolver))
+                    .flatMap(pathModel -> liveCopyService.getResourcePathWithLiveCopiesPaths(resourceResolver, pathModel.getPath(), pathModel.isIncludeLiveCopies())
```

Comment:
(cosmetic) Consider moving the part after `pathModel ->` to a separate method. It is a Stream API chain within a Stream API chain == always reads with difficulty

Reply:
done